doc-content/shared-kie-docs/src/main/asciidoc/Workbench/RemoteAPI/security-management-rest-api-supported-permissions-ref.adoc

```diff
@@ -0,0 +1,99 @@
+[id='security-management-rest-api-supported-permissions-ref_{context}']
+= Supported permissions in {CENTRAL}
+
+The following are available permissions in {PRODUCT}. Administrators use these permissions to allow specific actions to a group, role, or user in {CENTRAL}.
+
+Priority::
+Priority is an integer that defines the precedence of users who are assigned multiple roles or groups. The default value of priority for a new group is `-100`. In {CENTRAL}, you can set an integer value as a priority, which is resolved using the following rules:
++
+--
+.Priority value table
+[cols="50%,50%", frame="all", options="header"]
+|===
+|Integer value
+|Priority
+
+|Between -5 and 0
+|LOW
+
+|Less than -5
+|VERY LOW
```

Comment:
Very low should be the first in the table I think.

Reply:
+1

Reply:
Done